DataHub/UserInterface/08_VariantSets.md

```diff
@@ -0,0 +1,338 @@
+
+- [DELETE]     
+    Click this button to delete the selected variant sets. A variant set can only be deleted if no dependencies are existing. Otherwise, an error message is displayed. This button is only displayed if the checkbox of at least one variant set is selected.
+
+The following functions and fields are available in this view:
+
+- *Name*   
+    Variant set name.
+
+- *Attribute set*   
+    Attribute set name assigned to the variant set.
+
+- *Attributes (defining)*   
+    Defining attributes of the variant set.
+
+- *Attributes (changeable)*   
+    Changeable attributes of the variant set.
+
+- *ID*   
+    Variant set identification number. The ID number is automatically assigned by the system.
+
+- *Modified on*   
+    Date and time of the last modification.
+
+- *Modified by*   
+    Name and username of the user who modified the variant set.
+
+- *Created on*   
+    Date and time of the creation.
+
+- *Created by*   
+    Name and username of the user who created the variant set.
+
+- *Name (Language)*   
+    Variant set name in the selected language. A single column is displayed for the system languages *English (United States)* and *Deutsch (Deutschland)*.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
+    Click this button to create a variant set. The *Create variant set* window is displayed.   
+
+
+
+## Create variant set
+
+*PIM > Settings > Tab VARIANT SETS > Button Add*  
+*Omni-Channel > Settings > Tab VARIANT SETS > Button Add*
+
+![Create variant set](../../Assets/Screenshots/DataHub/Others/CreateVariantSet.png "[Create variant set]")
+
+**Create variant set**
+
+The following functions and fields are available in this view:
+
+- *"Language name"*   
+    Click the drop-down list and select the language in which the *Name (Language)* fields are displayed. The languages *English (United States)* and *Deutsch (Deutschland)* are available in the drop-down list.
+
+- [SAVE]   
+    Click this button to save the variant set. The *Create variant set* view is closed.
+
+- *Name (Language)*   
+    Variant set name in the selected language.
+
+- *Attribute set*   
+    Click the drop-down list to select the assigned attribute set. All attribute sets are displayed in the drop-down list.
+
+**FORMULAS**
+
+This section is only displayed if an attribute set is selected.
+
+- *Formula for SKU*   
+    Click the field to edit the formula for the SKU of the variant articles. By default, the formula **{master}-{L}** is predefined. The following placeholders are available:
+    - **{master}**   
+        SKU number of the master entity
+    - **{L}**   
+        consecutive number   
+    
+    > [Info] In addition to these default placeholders, you can include the value of the defining attributes of the variant set into the SKU formula. The corresponding placeholders are displayed in the *Possible placeholders* section when you have added a defining attribute.
```

Comment:
Die Info hier wird komisch angezeigt in der "normalen" Ausgabe. Vielleicht einfach im einführenden Satz unterbringen?
![image](https://github.com/Actindo-AG/Documentation/assets/133089306/e1da56f6-02a2-4be7-94e8-63e329f02aff)